app/aggregates/todo.rb

```diff
@@ -3,8 +3,15 @@ module Aggregates
     class Todo
       include EventSourcery::AggregateRoot
 
+      # These apply methods are the hook that this aggregate uses to update
+      # its internal state from events.
+
       apply TodoAdded do |event|
-        @added = true
+        # We track our ID when we are added to we can ensure the same todo isn't
```

Comment:
`We track our ID when we are added to we can/We track the ID when the event todo is added to`